MapboxNavigation/FeedbackViewController.swift

```diff
@@ -53,12 +52,19 @@ class FeedbackViewController: UIViewController, DismissDraggable, UIGestureRecog
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        
+        //FIXME: This is a workaround to ensure that the FVC looks good on non 375pt width screens, as the dynamic sizing logic isn't currently working properly.
+        flowLayout.itemSize = collectionView(collectionView, layout: flowLayout, sizeForItemAt: IndexPath(item: 0, section: 0))
```

Comment:
should we probably use **size classes** to set the `itemSize` for specific screen sizes?

Reply:
However, if we choose to go with this route, could we consider using the `UICollectionViewDelegateFlowLayout` functions? Like so:
```
// For size
func collectionView(UICollectionView, layout: UICollectionViewLayout, sizeForItemAt: IndexPath)
```

Reply:
This approach isn't possible because `375`pt width screens (X, 8, 7, 6) share the same size class (`wChR`) as `320`pt width (iPhone SE) screens. Ideally, a `UICollectionViews` auto-size cell logic would take care of variations in the same size-class, but it's not working correctly. That's what this workaround addresses.

Reply:
Hmmm... so weird tho. Got it.

Reply:
Yeah, It's an artifact of our "half-storyboard, half-code" UI state. As we continue moving everything into code, `sizeForItemAtIndexPath:` should start giving us correct values again.

Reply:
@JThramer Yeah, I was having this conversation with @bsudekum earlier on. It makes more sense to keep things mostly in storyboards if we can. To address the potential merge-conflicts we could split the main storyboards into connected storyboards to handle specific tasks. (In case of merge-conflicts concern, I can definitely assist in this domain. 😸 ) 
cc @mapbox/navigation-ios

Reply:
Replied to [here.](https://github.com/mapbox/mapbox-navigation-ios/pull/1063#issuecomment-367859900)